Formula/rpki-client.rb

```diff
@@ -16,16 +16,22 @@ class RpkiClient < Formula
   end
 
   depends_on "pkg-config" => :build
-  depends_on "libressl"
   depends_on "rsync"
+
   uses_from_macos "expat"
 
+  on_macos do
+    depends_on "libressl"
+  end
+  on_linux do
+    depends_on "libretls"
+    depends_on "openssl@3"
```

Comment:
Not sure about macOS (since based on BSD), but probably should just use `openssl` on Linux. Also avoids duplicate library due to `rsync --> openssl@3`.

Reply:
Looks like OpenSSL gets picked up on macOS. Might as well do this on macOS like MacPorts https://ports.macports.org/port/rpki-client/
```
Homebrew libraries:
  /usr/local/opt/libressl/lib/libtls.25.dylib (libressl)
  /usr/local/opt/openssl@3/lib/libcrypto.3.dylib (openssl@3)
Indirect dependencies with linkage:
  openssl@3
```